app/_src/gateway/plugin-development/custom-logic.md

```diff
@@ -42,7 +42,7 @@ of {{site.base_gateway}}'s execution life-cycle:
 - **[HTTP Module]** *is used for plugins written for HTTP/HTTPS requests*
 {% if_version lte: 3.3.x %}
 
-| Function name       | Phase             | Request Protocol        | Description
+| Function name       | Directive             | Request Protocol        | Description
```

Comment:
I think it's a bit more complicated than this. The items in this column cover both a phase and a directive. Check out the diagram here: https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#directives.



As the reporter points out in the GH issue,
* "rewrite" in Kong -> stands for the "rewrite tail" phase and the "rewrite_by_lua_*" directive: https://github.com/openresty/lua-nginx-module#rewrite_by_lua_block

It looks like what we're actually missing is another column, which details what the directive is. So for rewrite, you get:

Function: rewrite
Phase: rewrite tail
Directives: `rewrite_by_lua_*`

Reply:
Did a bunch more digging, and this isn't quite right either. It seems that Kong has come up with its own terminology here, where "access phase" or "init_worker phase" are accurate _only for Kong_. So - technically this column _is_ correctly named "phase", but the links to directives make it confusing.